libfastpm/lightcone-smesh.c

```diff
@@ -0,0 +1,751 @@
+#include <math.h>
```

Comment:
This file has been removed from master:

https://github.com/rainwoodman/fastpm/search?q=smesh&type=Commits